### ISSUE TITLE
Question on code

### ISSUE THREAD

**Issue by houjunlin** [seefun/Stochastic_Label_Refinery#1]
Thank you for the excellent work! Could you share the code of this paper?

Thanks again!

**Comment by seefun** [comment]
Thank you for your attention and the question!
Because the method itself is relatively easy to re-produce, It can be really easily used in a variety of tasks (Not only in DR dataset).
In order to better show the usage of the method itself, I put the code of our toy-experiment [here](https://github.com/seefun/Stochastic_Label_Refinery/tree/master/toy_exp)
If you need the original code on the DeepDR data set, It may take some time to clean the code style (I have to find the code two years ago first).

**Comment by seefun** [comment]
You can find the original code and training log (train in official training set and test in official validation set) in DeepDR dataset from the paper [here](https://github.com/seefun/Stochastic_Label_Refinery/tree/master/deepdr-original-code).
Note that the SOTA result on this dataset is achieved by merging training and validation set to train SLR models, and test in official testing set.

**Comment by houjunlin** [comment]
It's very kind of you to give such a detailed reply. It's really helpful! Thank you so much! :)

**Comment by seefun** [comment]
Thank you again for your attention to this two-years-ago work!
SLR is a powerful tool to achieve SOTA in huge amount of dataset (add SLR on existing SOTA method). And I used it to get great results of many competitions. I hope you can get help from it!
